### DISCUSSION TITLE
When will we release InternLM2 1.8B[Feature]

### DISCUSSION THREAD

**Discussion by LYMDLUT** [InternLM/InternLM#685] [Q&A] [answered]
### 描述该功能

As we all know,1.8B could freely use on more devices with low memory GPU,when will we release it?

It is easy to use, easy to learn. Time used to load models will decrease sharply, it will be more friendly infering on mobile devices and even could be combined with Clip to be  a multimodel assistant.

**Comment by vansin** [comment]
血书开源 InternLM2 1.8B

**Comment by JimmyMa99** [comment]
血书开源 InternLM2 1.8B

**Comment by matrixgame2018** [comment]
血书开源 InternLM2 1.8B

**Comment by KMnO4-zx** [comment]
血书开源 InternLM2 1.8B

**Comment by Jianfeng777** [comment]
血书开源 InternLM2 1.8B！

**Comment by melonwine** [comment]
支持开源InternLM2上手机！

**Comment by hu-qi** [comment]
hurry up!

**Comment by YanxingLiu** [comment]
血书开源 InternLM2 1.8B

**Comment by maxchiron** [comment]
血书开源 InternLM2 1.8B 
![image](https://github.com/InternLM/InternLM/assets/85681332/953b0d6f-502f-4575-9feb-650cb415ddcd)

**Comment by isLinXu** [comment]
血书开源 InternLM2 1.8B

**Comment by 443752009** [comment]
血书开源 InternLM2 1.8B

**Comment by yhcc** [comment]
在训了在训了，会开源的

**Comment by Howardzhangdqs** [comment]
血书开源 InternLM2 1.8B

**Comment by Hongru0306** [comment]
血书开源 InternLM2 1.8B

**Comment by MRQJsfhf** [comment]
不是有11亿与13亿参数的吗?只是训练数据80%及以上都是代码数据。不过性能还行

**Comment by zRzRzRzRzRzRzR** [comment]
期待一手！

**Comment by 00INDEX** [comment, marked as answer]
InternLM2-1.8B has been open-sourced.
internlm/internlm2-1_8b: https://huggingface.co/internlm/internlm2-1_8b
internlm/internlm2-chat-1_8b-sft: https://huggingface.co/internlm/internlm2-chat-1_8b-sft

**Reply by LYMDLUT** [reply]
Many thanks to our great InternLM team！

**Reply by zRzRzRzRzRzRzR** [reply]
wow，牛啊，感谢开源贡献！
